index.php

```diff
@@ -0,0 +1,51 @@
+
+<!doctype html>
+<html lang="en">
+<head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>Wine</title>
+
+</head>
+<body>
+<div class="titleBar">
+    <h1>Let's make some pour decisions!</h1>
+</div>
+
+<div class="container">
+    <?php
+    $db = new PDO('mysql:host=db;dbname=wineCollection', 'root', 'password'); //initialise the db connection
+    $db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
+    $query = $db->prepare('SELECT `variety`, `tones`, `cost`, `nameOfBrand`, `country`, `wine`.`img_location` FROM `wine` JOIN `brands` ON `wine`.`brand_id` = `brands`.`id` JOIN `country` ON `wine`.`country_of_origin`=`country`.`id`;');
+    $query->execute();
+    $wines=$query->fetchAll();
+
+    foreach($wines as $wine){
```

Comment:
mike also says all this looping can be contained in a function. again, i haven't done it yet either!